lib/workos/types/passwordless_session_struct.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+# typed: strict
+
+module WorkOS
+  module Types
+    # This PasswordlessSessionStruct acts as a typed interface
+    # for the Passwordless class
+    class PasswordlessSessionStruct < T::Struct
+      const :id, String
+      const :email, String
+      const :expires_at, Date
+      const :link, String
+      const :object, String
```

Comment:
I think we can leave off `object` this is metadata from the API we don't need to expose

Reply:
- [ ] remove object metadata field